lib/inspec/utils/telemetry.rb

```diff
@@ -17,6 +18,15 @@ def self.determine_backend_class
       # Don't perform telemetry call if license is not a free license
       return Inspec::Telemetry::Null unless license&.license_type == "free"
 
+      if Inspec::Dist::EXEC_NAME == "inspec" && telemetry_disabled?
+        # Issue a warning if an InSpec user is explicitly trying to opt out of telemetry using cli option
+        Inspec::Log.warn "Telemetry opt-out is not permissible."
+      end
+
+      # Telemetry opt-in/out enabled for other Inspec distros
+      return Inspec::Telemetry::Null if Inspec::Dist::EXEC_NAME != "inspec" && telemetry_disabled?
+      # TODO: ask chef-telemetry gem if we should be enabled or not
```

Comment:
Not understtood what is this TODO is for.

Reply:
Removed